website/cue/reference/components/sinks/base/greptimedb_metrics.cue

```diff
@@ -99,6 +99,18 @@ base: components: sinks: greptimedb_metrics: configuration: {
 		required: false
 		type: string: examples: ["grpc_compression"]
 	}
+	new_naming: {
```

Comment:
When/why would people enable this option? The description doesn't make it super clear.